azure-pipelines-full.yml

```diff
@@ -0,0 +1,11 @@
+# Run manually or by schedule only.
+trigger: none
```

Comment:
Can the full be run using a keyword in either a PR message or a commit message?

Reply:
It now runs only when merged to master, the idea being that a manual trigger could be used for PRs via commenting. I tried it out here but it didn't work, not sure exactly why. I'll ask someone tomorrow who may know, but alternatively we could also run it always for PRs.

Reply:
The alternatively to always run for PRs is good with me. We would not expect to merge in so many PRs in the WRF-CMake project.

Reply:
Opted for the run-always variant for now...

Reply:
Agree